src/mon/Monitor.cc

```diff
@@ -2283,9 +2283,8 @@ void Monitor::health_tick_start()
       if (r < 0)
         return;
       do_health_to_clog();
-      health_tick_start();
-    });
-  timer.add_event_after(cct->_conf->mon_health_to_clog_tick_interval,
+  });
+  timer.add_cycle_event_after(cct->_conf->mon_health_to_clog_tick_interval,
                         health_tick_event);
```

Comment:
Use common/ceph_timer.h instead

If you want to add a cycle event interface to that you can, it should be fairly easy.

(Just use the already existing reschedule support.)

Or just use reschedule_me in the caller.